go/service/account.go

```diff
@@ -276,14 +277,49 @@ func (h *AccountHandler) TimeTravelReset(ctx context.Context, arg keybase1.TimeT
 	return err
 }
 
-func (h *AccountHandler) UserGetContactSettings(ctx context.Context) (ret keybase1.ContactSettings, err error) {
+func (h *AccountHandler) UserGetContactSettings(ctx context.Context) (res keybase1.ContactSettings, err error) {
 	mctx := libkb.NewMetaContext(ctx, h.G())
 	defer mctx.TraceTimed("AccountHandler#UserGetContactSettings", func() error { return err })()
-	return libkb.GetContactSettings(mctx)
+	res, err = libkb.GetContactSettings(mctx)
+	for i, team := range res.Teams {
```

Comment:
tell me if you think this for loop will be too expensive for people in large teams. on the other hand, @cjb is probably doing team id->name resolution in the frontend, so the frontend could just use the resolutions that the server makes here.

Reply:
This is probably fine as long as teh RPC is called only when the user is viewing that screen. This list is probably very small..